mycore-pi/src/test/java/org/mycore/pi/MCRPIUtils.java

```diff
@@ -74,7 +74,7 @@ public static URL getUrl(MCRPIRegistrationInfo info) {
         try {
             return new URI(url).toURL();
         } catch (MalformedURLException | URISyntaxException e) {
-            e.printStackTrace();
+            LOGGER.error(e.getMessage(), e);
             LOGGER.error("Malformed URL: {}", url);
```

Comment:
Logging twice is not necessary (and possibly confusing when reading the log), maybe simplify: sth. like `LOGGER.error("Malformed URL: " + url, e);`